packages/donor/src/components/Wizard/WizardScreen.module.scss

```diff
@@ -0,0 +1,26 @@
+:root {
```

Comment:
What does `:root` do? Does this affect the entire app?

Reply:
changed it to  .container